packages/atoms/src/components/kiwi-toasts/kiwi-toasts.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component, Host, h, Listen, State } from '@stencil/core';
+import { map } from 'lodash-es';
+
+export interface Toast {
+  title: string;
+  description: string;
+  type: 'info' | 'warn' | 'error' | 'success';
+  timeout?: number; // 0 means no automatic disappearing
+}
+
+const DEFAULT_DURATION = 5000;
+
+@Component({
+  tag: 'kiwi-toasts',
+  styleUrl: 'kiwi-toasts.css',
+  shadow: false,
+})
+export class KiwiToasts {
+  @State()
+  private toasts: { [key: string]: Toast } = {};
```

Comment:
Bei einem Objekt ist die Reihenfolge glaube ich nicht garantiert. Hier wäre ein Array vermutlich die bessere Wahl um evtl. Bugs zu vermeiden. Du könntest den key in das Toast interface ziehen und das Löschen über `reject(this.toasts, { key })` realisieren.

Reply:
Ich würde es gerne vermeiden, dass man sich als Entwickler einer Anwendung Gedanken über den Key eines Toasts (den man gerade schmiert) machen muss. Deswegen generiere ich den im Eventhandler.
Dann nehme ich lieber eine Map, da dort die Iterationsreihenfolge konstant ist: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Objects_vs._Maps

Reply:
Ach stimmt, Immutabilität für Change Detection. Ich geh dann mal meinen Code nochmal ändern (und verwende jetzt ein Array) :-|